x/ibc/20-transfer/keeper/relay.go

```diff
@@ -161,13 +163,15 @@ func (k Keeper) createOutgoingPacket(
 			coins[i] = sdk.NewCoin(coin.Denom[len(prefix):], coin.Amount)
 		}
 
+		// escrow source tokens (assumed to fail if balance insufficient)
```

Comment:
"assumed" is scary here. Is there a test for that? Do you mean "fails if balance is insufficient"?

Reply:
Agreed. I think this leads into a discussion of acknowledgement packets for this module, something I know @cwgoes and I have discussed.

Reply:
the transaction fails if balance is insufficient. I'll remove the "assumed"